CCatHome_Regional_Host_Guide.md

```diff
@@ -0,0 +1,228 @@
+# Regional Host, Co-host/CoC Facilitators and Speakers Guide
+
+This is the guide for CarpentryCon\@Home Presenters and Regional Guides.
+
+## General Information
+
+### Video Conferencing
+
+CarpentryCon\@Home will be hosted via the [Zoom Conferencing Platform](https://zoom.us/) and [crowdcast.io](https://www.crowdcast.io/)
+
+In the agenda you will see the links to either Zoom or crowdcast events.
+
+#### Zoom accounts:
+
+- Create a free zoom account: [https://zoom.us/](https://zoom.us/)
+- Test that your headphones and webcam works: [https://zoom.us/test](https://zoom.us/test)
+
+- Follow the link to the online agenda where the zoom links for registration should be available for each session
+
+## Important Guidelines
+
+1. [Quick links and emergency contacts](https://docs.google.com/document/d/1VCDLt72aOwle9xTYzhbs4CHhnaZCvEzwLne5xb0QjLg/edit?usp=sharing)
+
+2. [Understanding your role as a host, co-host and CoC facilitator](https://carpentryconnect.org/online/#DefiningVolunteerRoles)
+
+3.  Useful links and guidelines for everyone:
+    - [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing) in details
+
+    - Crowdcast Instruction (coomming soon)
+
+## Code of Conduct
+
+All hosts, speakers, and attendees of the CarpentryCon\@Home are
+Expected to adhere to the Carpentries Code of Conduct.
+
+- [Code of conduct: online documents](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
+
+- [Code of conduct shared folder](https://drive.google.com/drive/folders/1iQX5TFWuXmeXERgqR4zI0QaAulEj_6OD?usp=sharing)
+
+- Direct link to the [CoC Facilitators Roles and Responsibilities](https://docs.google.com/document/d/1p0GIrNzXHYJlJmFr_18oOkOIFRnfH51BqlBmqhNni-Y/edit?usp=sharing)
+
+## Role-Specific Guidelines for CarpentryCon@Home
+
+There are following roles in the CarpentryCon\@Home event beside the
+task force members:
+- Regional hosts
+- CoC facilitators who will be present as co-hosts with the regional hosts
```

Comment:
I'm not sure we're ready for this. Can this be removed for now and a note added above that there will be at least a CoC contact person that people can email?

Reply:
I agree. We thought maybe we would have specific guides for this.

Reply:
I removed it.

Reply:
We do have facilitator document, correct me if that is not what you are talking about.

Reply:
This can stay in ! I just wanted ot make sure we had a plan for getting facilitators which Malvika clarified this morning.

Reply:
Ok, I'm putting it back